utHelper.py

```diff
@@ -0,0 +1,372 @@
+            destinationPath (str): The destination path and filename on the device.
+            device (str, optional): The device to copy the file to (default: "dut").
+
+        Returns:
+            str: The message from the subprocess (SCP output).
+
+        Raises:
+            ValueError: If the session type is not "ssh".
+        """
+        activeDevice = self.devices.getDevice(device)
+
+        if activeDevice.session.type == "ssh":
+            self.log.stepMessage("copyFile(" + sourcePath + ", (" + destinationPath + ")")
+
+            # TODO: Assumption: 'root' user is the default on the target, this should be specified by the configuration
+            destination = "root@{}:{}".format(self.slotInfo.getDeviceAddress(), destinationPath)
+
+            # Construct the SCP command with options to disable strict host key checking and known_hosts file
+            command = ["scp", "-o", "StrictHostKeyChecking=no", "-o", "UserKnownHostsFile=/dev/null",
+                    sourcePath, destination]
+
+            # Execute the SCP command and capture the output
+            result = subprocess.run(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+            message = result.stdout.decode('utf-8').strip()
+
+        elif self.session.type == "serial":
+            # self.writeMessageToDeviceSession("cp " + source + " " + destination)  # Commented out code, potentially for serial copy
+            self.log.error("Can't copy for this session type: serial")
+            raise ValueError("Copying files is not supported for serial connections.")
+
+        return message
+
+
+# Device / Host Command operations
+
+    def writeHostCommand(self, command):
+        """
+        Executes a command on the host machine (Linux only).
+
+        Args:
+            command (str): The command to execute.
+
+        Returns:
+            str: The output/result of the command execution.
+        """
+        self.log.debug("writeHostCommand()")
+        result = self.syscmd(command)  # Execute the command using syscmd (assuming it's defined elsewhere)
+        return result
+
+    def flushSessionData(self, timeout=1, device="dut"):
+        """
+        Clears any pending data from the session buffer of the specified device.
+
+        Args:
+            timeout (int, optional): The timeout in seconds to wait before reading (default: 1).
+            device (str, optional): The device to flush the session data for (default: "dut").
+
+        Returns:
+            str: The accumulated data that was read from the session buffer.
+        """
+        self.log.debug("clearCommand()")
+        self.waitSeconds(timeout)  # Wait for a short time to allow data to accumulate
+
+        activeDevice = self.devices.getDevice(device)
+        session = activeDevice.getConsoleSession()
+        result = session.read_all()  # Read all available data from the session
+        return result
```

Comment:
I'm not sure if this will work with any other console type than SSH. And SSH's read functions are changing with Fatmas changes so we may want to hold off on this.